### PR TITLE
Adds package-lock.json

### LINKED ISSUE #206: Failure to build JS on clean installation
**Describe the bug**
In a clean checkout, the JS code will not compile.

**To Reproduce**
```bash
git clone https://github.com/exa-analytics/exatomic.git
cd js
./npminstall.sh
```

**Expected behavior**
The code should compile.

**Desktop (please complete the following information):**
 - OS: WSL Ubuntu 20.04 LTS
 - Version 0.5.2

**Additional context**
This does not occur for a local build which contains an uncommitted `package-lock.json` file. Suspect the current `package.json` is missing pinned dependencies or nested dependencies.

### PR DESCRIPTION
**Pull request by tjduigna**
Closes #206

### CODE REVIEW

File: exatomic/algorithms/orbital_util.py [hunk 1]
```diff
@@ -243,6 +243,7 @@ def _determine_vector(uni, vector, irrep=None):
         else:
             uni.atom['Z'] = uni.atom['symbol'].map(sym2z)
             homo = uni.atom['Z'].sum() // 2
+        homo = int(homo)
```
Comment (avmarchenko) [inline exatomic/algorithms/orbital_util.py:246]: Sneaking this in eh ;) ? Does this close an open issue? (Likewise elsewhere)

**Review by avmarchenko — APPROVED** [review]
Thank you for responding to the issue so quickly.

File: exatomic/algorithms/orbital_util.py (hunk 1 above)
Comment (avmarchenko) [reply exatomic/algorithms/orbital_util.py:246]: If it is for the Appv build, thanks. I think we can safely ignore the win32 builds on appv (they will likely fail for other reasons). If we don't already have it, we should create an issue to deprecate those builds - I don't think any users of ex/atomic are running win32.

**Review by herbertludowieg — APPROVED** [review]

File: exatomic/algorithms/orbital_util.py (hunk 1 above)
Comment (tjduigna) [reply exatomic/algorithms/orbital_util.py:246]: It was causing test failures in travis so it needed to happen. +1 to deprecating win32 builds though.

**Pull request merged**